examples/thermodynamics/energy_from_combustion_of_gasoline.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+from sympy import solve, Symbol, Eq
+from symplyphysics import print_expression
+from symplyphysics.definitions import density_from_mass_volume as density_law
+from symplyphysics.laws.kinematic import distance_from_constant_velocity as distance_law
+from symplyphysics.laws.electricity import power_factor_from_active_and_full_power as efficiency_law
+from symplyphysics.laws.electricity import power_from_energy_time as power_law
+from symplyphysics.laws.thermodynamics import energy_from_combustion as combustion_energy_law
+
+# Example from https://easyfizika.ru/zadachi/termodinamika/na-skolko-kilometrov-puti-hvatit-40-l-benzina-avtomobilyu-dvizhushhemusya-so-skorostyu/
+# How many kilometers will 40 liters of gasoline be enough for a car moving at a speed of 54 km/h?
+# The power developed by the car engine is 46 kW, and the efficiency is 25%.
+
+volume_of_gasoline = Symbol("volume_of_gasoline")
+velocity_of_car = Symbol("velocity_of_car")
+efficiency_factor = Symbol("efficiency_factor")
+power_of_car = Symbol("power_of_car")
+
+density_of_gasoline = Symbol("density_of_gasoline")
+gasoline_specific_heat_combustion = Symbol("gasoline_specific_heat_combustion")
+
+distance = Symbol("distance")
+
+velocity_equation = distance_law.law.subs({
+    distance_law.distance(distance_law.movement_time): distance,
+    distance_law.constant_velocity: velocity_of_car,
+    distance_law.initial_position: 0
+})
+time_equation = solve(velocity_equation, distance_law.movement_time, dict=True)[0][distance_law.movement_time]
```

Comment:
It's value, not equation. Also check other places, where you use `equation` when it's not.